app/models/grda_warehouse/tasks/push_clients_to_cas.rb

```diff
@@ -25,10 +24,10 @@ def sync!
       if GrdaWarehouse::DataSource.advisory_lock_exists?(advisory_lock_key)
         msg = 'Other CAS Sync in progress, exiting.'
         logger.warn msg
```

Comment:
Need to check the remaining file too
```suggestion
        Rails.logger.warn msg
```